CIP-0118/README.md

```diff
@@ -0,0 +1,1042 @@
+---
+CIP: 0118
+Title: Babel fees via validation zones
```

Comment:
Do we want to align the title of the pull request with this title?

Reply:
Yes... especially since as I understand it this proposal will enable applications of far wider range than just Babel Fees (@polinavino?).  The very general title should be fine for this CIP because it doesn't look like a "competing" proposal for Validation Zones would have such a good chance as this one of getting endorsement from the Ledger team.

Reply:
that make sense (will change it)!